### DISCUSSION TITLE
Windows x86?

### DISCUSSION THREAD

**Discussion by Git-Maniac** [c0re100/qBittorrent-Enhanced-Edition#630] [Q&A] [answered]
Since version 5.0, has there been no more x86 version for Windows to download?

I mean, the release 'qbittorrent_enhanced_5.0.2.10_setup.exe' is missing...

Thanks

**Comment by c0re100** [comment, marked as answer]
**Start from v5.0.0, qBittorrent only support Windows 10 64bit 1809+**
https://github.com/c0re100/qBittorrent-Enhanced-Edition/releases/tag/release-5.0.0.10
